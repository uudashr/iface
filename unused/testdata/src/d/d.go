package actor

// Doer performs some work.
//
//iface:ignore
type Doer interface {
	Do() error
}

type Greeter interface { // want "interface 'Greeter' is declared but not used within the package"
	Greet() error
}

//iface:ignore=unused
type Runner interface {
	Run() error
}

//iface:ignore=other
type Executor interface { // want "interface 'Executor' is declared but not used within the package"
	Execute() error
}

//iface:ignore=other,unused
type Server interface {
	Serve() error
}
