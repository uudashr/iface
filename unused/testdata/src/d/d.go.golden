package actor

// Doer performs some work.
//
//iface:ignore
type Doer interface {
	Do() error
}

//iface:ignore=unused
type Runner interface {
	Run() error
}

//iface:ignore=other

//iface:ignore=other,unused
type Server interface {
	Serve() error
}
